compiler/x/codegen/OMRX86LinkageProperties.hpp

```diff
@@ -0,0 +1,220 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
```

Comment:
Forward date should be 2020.

Reply:
Thanks for your time @0xdaryl! 

Will have this fixed in next commit.